source/index.html

```diff
@@ -1,12 +1,100 @@
+    <ul class="site-navigation__list site-nav-list">
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="catalog.html">Каталог продукции</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="index.html">Главная</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="form.html">Подбор программы</a></li>
+    </ul>
+  </nav>
+</header>
+<main class="page-main">
+  <section class="product-promotion">
+    <h1 class="product-promotion__headline main-headline main-headline--white">Функциональное питание для котов</h1>
+    <p class="product-promotion__slogan">Занялся собой? Займись котом!</p>
+    <img class="product-promotion__image" src="#" alt="Коробка с питанием для котов и кошек Cat Energy Pro.">
+    <a class="product-promotion__link order-button" href="form.html">Подобрать программу</a>
+  </section>
+  <section class="weight">
+    <h2 class="weight__headline cat-weight-headline">Похудение</h2>
+    <p class="weight__promo">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.</p>
+    <a class="weight__link catalog-link-slim" href="catalog.html#500-slim">Каталог Slim</a>
+  </section>
+  <section class="weight">
+    <h2 class="weight__headline cat-weight-headline">Набор массы</h2>
+    <p class="weight__promo">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!</p>
+    <a class="weight__link catalog-link-pro" href="catalog.html#500-pro">Каталог pro</a>
+  </section>
+  <section class="principles">
+    <h2 class="principles__headline index-section-headline">Как это работает</h2>
+    <ul class="principles__list principles-list">
+      <li class="principles-list__item">Функциональное питание содержит только полезные питательные вещества.</li>
+      <li class="principles-list__item">Выпускается в виде порошка, который нужно лишь залить кипятком и готово.</li>
+      <li class="principles-list__item">Замените один-два приема обычной еды на наше функциональное питание.</li>
+      <li class="principles-list__item">Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</li>
+    </ul>
+  </section>
+  <article class="promo">
+    <h2 class="promo__headline index-section-headline">Живой пример</h2>
+    <p class="promo__story advert-story">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день. </p>
+    <div class="promo__div">
+      <p class="promo__text"><span class="promo__number">5 кг</span> <span class="promo__action-description">снижение веса</span>
+      </p>
+      <p class="promo__text"><span class="promo__number">60 дней</span> <span class="promo__action-description">затрачено времени</span>
```

Comment:
```suggestion
      <<span class="promo__span">затрачено времени</span>
```
ничтоже сумняшеся, я, обычно, так называю, потому что нельзя привязываться к содержимому